src/codemagic/tools/_app_store_connect/arguments.py

```diff
@@ -4,6 +4,9 @@
 from codemagic.apple.app_store_connect import AppStoreConnectApiClient
 from codemagic.apple.app_store_connect import IssuerId
 from codemagic.apple.app_store_connect import KeyIdentifier
+from codemagic.apple.app_store_connect.builds import Builds
```

Comment:
For some reason can't comment on previous ones

> Can't you reuse Builds.Ordering as CLI argument value insead?

I want to keep cli arguments consistently uppercase, while ordering is in camel

> I get the signature change, but instance attributes should be defined nonetheless. If attribute self is None, then it shouldn't just be added to dict. But in case the attributes are not defined entirely, this could have broad negative effect in many unforseen places.

right, it's better to not include `None` fields to `DictSerializable`'s `dict` method

> Could you elaborate what exactly is going on here?
```
In [9]: dt = datetime(2021, 2, 12)
In [10]: Resource.to_iso_8601(dt)
Out[10]: '2021-02-12T00:00:00'
In [11]: utc_dt = datetime(2021, 2, 12, tzinfo=timezone.utc)
In [12]: Resource.to_iso_8601(utc_dt)
Out[12]: '2021-02-12T00:00:00.000+0000'
```

So that both string date formats would be converted back to string from datetime to their original form
```python
>>> s_utc = '2020-08-04T11:44:12.000+0000'
>>> dt_utc = datetime.strptime(s_utc, '%Y-%m-%dT%H:%M:%S.%f%z')
>>> dt_utc
datetime.datetime(2020, 8, 4, 11, 44, 12, tzinfo=datetime.timezone.utc)
>>> dt_utc.strftime('%Y-%m-%dT%H:%M:%S.%f')[:-3] + '+0000'
'2020-08-04T11:44:12.000+0000'
>>>
>>> s_timedelta = '2021-01-28T06:01:32-08:00'
>>> dt_timedelta = datetime.strptime(s_timedelta, '%Y-%m-%dT%H:%M:%S%z')
>>> dt_timedelta
datetime.datetime(2021, 1, 28, 6, 1, 32, tzinfo=datetime.timezone(datetime.timedelta(days=-1, seconds=57600)))
>>> dt_timedelta.isoformat()
'2021-01-28T06:01:32-08:00'
```